### ISSUE TITLE
Game mode - Specific skin(s)

### ISSUE THREAD

**Issue by Zemanzo** [Zemanzo/web-marbles#95]
**Description**
All marbles use the same skin

**Minimum Acceptance Criteria**
- [ ] Player skin choices are ignored and a random specific skin is applied to all marbles
- [ ] Game mode will apply for a single round

**Stretch goal**
Sets of skins are applied (e.g. all planets in the solar system), players are still able to pick from this specific set.

**Is blocked by the following issues**
#137
#296
